src/components/common/NotFound/NotFound.tsx

```diff
@@ -1,8 +1,8 @@
-import { FC } from "react";
 import { P1 } from "@deskpro/deskpro-ui";
+import type { FC, ReactNode } from "react";
 
 export type Props = {
-  text?: string,
+  text?: ReactNode,
```

Comment:
Would be better named as `children` so it can be used like `<NotFound>No burgers found!</NotFound>`

Reply:
We use this kind of interface in all our apps. It would be better to be consistent everywhere.